FlowCrypt/Controllers/Threads/ThreadDetailsViewController.swift

```diff
@@ -156,6 +156,7 @@ extension ThreadDetailsViewController {
             : []
 
         let subject = input.rawMessage.subject ?? "(no subject)"
+        let threadId = quoteType == .reply ? input.rawMessage.threadId : nil
```

Comment:
Let's test by `!= .forward`, then you don't have to adjust it (or forget to adjust it) once reply-all is added.

Reply:
But I think `quoteType` for reply-all will still be `reply`, but `recipients` array will contain more emails

Reply:
Got it. I was just suggesting blindly without context. All good.